<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OFullKeyboard</class>
 <widget class="QFrame" name="OFullKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <widget class="QFrame" name="frameBorder">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutBorder">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frameHeader">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayoutHeader">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelType">
           <property name="text">
            <string>EN--cap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditShow">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonClose">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButton1">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton2">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton3">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton4">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton6">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton7">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton8">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton9">
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton0">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonLeft">
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonRight">
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButtonQ">
          <property name="text">
           <string>q</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonW">
          <property name="text">
           <string>w</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonE">
          <property name="text">
           <string>e</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonR">
          <property name="text">
           <string>r</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonT">
          <property name="text">
           <string>t</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonY">
          <property name="text">
           <string>y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonU">
          <property name="text">
           <string>u</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonI">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonO">
          <property name="text">
           <string>o</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonP">
          <property name="text">
           <string>p</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonDelete">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButtonA">
          <property name="text">
           <string>a</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonS">
          <property name="text">
           <string>s</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonD">
          <property name="text">
           <string>d</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonF">
          <property name="text">
           <string>f</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonG">
          <property name="text">
           <string>g</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonH">
          <property name="text">
           <string>h</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonJ">
          <property name="text">
           <string>j</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonK">
          <property name="text">
           <string>k</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonL">
          <property name="text">
           <string>l</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonSemicolon">
          <property name="text">
           <string>;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonEnter">
          <property name="text">
           <string>Enter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButtonZ">
          <property name="text">
           <string>z</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonX">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonC">
          <property name="text">
           <string>c</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonV">
          <property name="text">
           <string>v</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonB">
          <property name="text">
           <string>b</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonN">
          <property name="text">
           <string>n</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonM">
          <property name="text">
           <string>m</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonComma">
          <property name="text">
           <string>,</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonPeriod">
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonQuestion">
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonCap">
          <property name="text">
           <string>cap</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButtonPlus">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonMinus">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonAsterisk">
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonSlash">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonEqual">
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonExclam">
          <property name="text">
           <string>!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonSpace">
          <property name="text">
           <string>Space</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonCh_En">
          <property name="text">
           <string>EN</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
